scripts/ejpcsv2json.py

```diff
@@ -30,6 +30,13 @@ def first(lst):
     except IndexError:
         return None
 
+def unescape_html_escaped_values(row):
+    "mutator"
+    for key, val in row.items():
+        if isinstance(val, str):
+            row[key] = html.unescape(val)
```

Comment:
Personally prefer functional over mutator function. Might be okay in a small script like this one.

@giorgiosironi @GiancarloFusiello ?

Reply:
could be copied over to a new dictionary, as a mutator returning the result is misleading (if it's mutating it why returning it at all?)

Reply:
> Personally prefer functional over mutator

opposite of mutable would immutable, not functional.